src/alire/alire-origins-deployers-system-homebrew.adb

```diff
@@ -0,0 +1,273 @@
+with AAA.Strings; use AAA.Strings;
+
+with Alire.OS_Lib.Subprocess;
+with Alire.Errors;
+
+with Ada.Directories;
+with Ada.Environment_Variables;
+
+with GNAT.Regpat;
+
+with GNATCOLL.JSON;
+
+package body Alire.Origins.Deployers.System.Homebrew is
+
+   pragma Warnings (Off);
+
+   --  Ada.Strings.Unbounded is use-visible via Alire.Origins.
+   use GNATCOLL.JSON;
+
+   Homebrew_Prefix : constant String
+     := Ada.Environment_Variables.Value ("HOMEBREW_PREFIX", "");
```

Comment:
Actually, I suspect that this code will only get called at all if the `Distribution` is `Homebrew`? in which case the question is moot (but I would have to change the implementation of `Already_Installed `to use the `Installed_Version` found by `Get_Info`, because my lazy implementation relies on knowing `Homebrew_Prefix`).

And the tests for `Homebrew_Present` would be redundant.